src/bin/server.rs

```diff
@@ -19,6 +19,8 @@ struct Properties {
     lifetime: String,
     database_path: String,
     storage_path: String,
+    local_distributed_address: String,
```

Comment:
`server_address` is used to process requests from clients
`local_distributed_address` is used to process requests from other servers
because the server's communication uses the `tonic` now

Reply:
Ok, you can comment that "it will merge in the future"

Reply:
it will merge in the future

Reply:
I mean you add comments to the code

Reply:
> Ok, you can comment that "it will merge in the future"

Good suggestion.

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/55987696/204590235-6fefe14e-a75c-4a41-8751-d4f250bbce6d.png)
Isn't that so